galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -20,6 +20,19 @@ class NamespaceSerializer(ModelSerializer):
             queryset=auth_models.Group.objects.all()
     )
 
+    def validate_group(self, field):
+        groups = [RH_PE_ACCOUNT_SCOPE]
+        for account in field:
+            if account == RH_PE_ACCOUNT_SCOPE:
```

Comment:
Could the RH_PE_ACCOUNT_SCOPE checking stuff move into galaxy_api.auth.models.GroupManager somehow?

Reply:
And/or, would it be useful to define a PEGroup(Group) as a proxy model (https://docs.djangoproject.com/en/3.0/topics/db/models/#proxy-models) ?

Since the db schema is the same and should be the same table, but the app needs to
treat it a little different.